lerobot/scripts/train.py

```diff
@@ -175,19 +175,19 @@ def log_train_info(logger: Logger, info, step, cfg, dataset, is_online):
     num_episodes = num_samples / avg_samples_per_ep
     num_epochs = num_samples / dataset.num_samples
     log_items = [
-        f"step:{format_big_number(step)}",
```

Comment:
I think @Cadene will have an opinion about this. It's about trying to fit the output on one line. And I think I agree with him. I'm not sure the clarity is worth the ugliness of having this spill over onto two lines.

Reply:
Emmm @Cadene do you have spatial reasons to fit the output on one line? For example, you are developing a hardware with a very small screen, and you must fit all output within the screen?

Here is my story. I got stuck twice when I started training because:

1. The first time was the loooong data loading time because I didn't know what `data_s` meaned in the output log. If the log showed it was `dataloading time` more clearly, then I would know it was a dataloading problem and I could change `num_workers`, `batch_size`, etc. (I modified `batch_size` because I found the GPU utilization was too low
2. And then I found the looooong evaluation time because I didn't know I was using a sync env, instead of an asynv env. The logging just got stuck again...
3. Then I guessed perhaps I evaluated too many episodes. Thus, I set `eval.n_episodes=1` but I still got stuck again. The reason is this [line](https://github.com/huggingface/lerobot/blob/03d647269eff6d76df14c84159adc7624a582762/lerobot/scripts/eval.py#L241): 

```
n_batches = n_episodes // env.num_envs + int((n_episodes % env.num_envs) != 0)
# with n_episodes==1, we still get n_batches = 0 + 1 = 1
# Since we use 50 synchonized environments, we still evaluate 50 episodes one by one, even if the user sets n_episodes=1
```

If you do not need to fit the output in a small screen, I think the clarity is worth the ugliness:

### For a beginner user, he/she may only have 20 minutes in a weekday coffee break to try out LeRobot. If LeRobot is set up, run painlessly, and the initial output can be easily understood within the 20 minutes, the user will think it is a good tool and spends more time trying it later.

Reply:
Regarding your point 2, I think we can have some logic like:

```python
if eval.n_episodes < eval.batch_size:
    eval.batch_size = eval.n_episodes
```

Reply:
Nice idea, I just add it! I also set `async_env=False` for xarm env

Reply:
"Unresolving" this as we still have not heard from @Cadene.

I hear what you are saying but I still don't think it's worth it. IMO generally anyone who's advanced enough to consider timing bottlenecks, should be advanced enough to look into the code and see what these abbreviations mean. Most people who are not that advanced, wouldn't really notice it if it were written in plain english.

Reply:
You are right, to make it easier to understand, `number_of_seen_samples: 64` is better. If the general idea is approved, I will modify it.

I still hope it's as easy to understand as possible ^^

Reply:
Instead of this change, I would prefer to update the [tutorial on `train.py`](https://github.com/huggingface/lerobot/blob/main/examples/4_train_policy_with_script.md), we should explain a typical output with some troubleshooting when data loading is high, etc.

We do this for the incoming `control_robot.py` tutorial:
<img width="849" alt="Screenshot 2024-08-14 at 11 07 19" src="https://github.com/user-attachments/assets/803b36c2-5b98-4640-8f91-c9dee85f56ab">

What do you think @alexander-soare ?